doc/visual-programming/source/widgets/visualize/pythagoreantree.rst

```diff
@@ -51,11 +54,25 @@ Pythagorean Tree can visualize both classification and regression trees. Below i
 Example
 -------
 
-In this example we demonstrate the difference between :doc:`Classification Tree Viewer<../classify/classificationtreeviewer>` and **Pythagorean Tree**. They both visualize :doc:`Classification Tree<../classify/classificationtree>` algorithm, but Pythagorean tree is much more concise, even for a small data set such as _Iris.tab_.
+The workflow from the screenshot below demonstrates the difference between 
+:doc:`Classification Tree Viewer<../classify/classificationtreeviewer>` and Pythagorean Tree. They can both visualize :doc:`Classification Tree<../classify/classificationtree>`, but Pythagorean visualization takes less space and is more compact, even for a small 
+`Iris flower <https://en.wikipedia.org/wiki/Iris_flower_data_set>`_
+data set. For both visualization widget, we have hidden the control area on the left by clicking on the splitter between control and visualization area.
 
 .. figure:: images/Pythagorean-Tree-comparison.png
 
+Pythagorean Tree is interactive: click on any of the nodes (squares) to select training data instances that were associated to that node. The following workflow explores these feature. 
+
+.. figure:: images/Pythagorean-Tree-scatterplot-workflow.png
+    :scale: 80
+    :align: center 
+
+The selected data instances are shown as subset in the scatter plot, are sent to the data table and examined in the box plot. We have used brown-selected data set in this example. The tree and scatter plot are shown below; the selected node in the tree has a black outline.
```

Comment:
> The selected data instances are shown as subset in the scatter plot, are sent to the data table and examined in the box plot.

The selected data instances are shown as a subset in the :doc:`Scatter Plot <../visualize/scatterplot>`, sent to the :doc:`Data Table <../data/datatable>` and examined in the :doc:`Box Plot <../visualize/boxplot>`.